components/form-group/src/test.tsx

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
-import { mount } from 'enzyme';
+import { render } from '@testing-library/react';
 
 import { FormGroupDocumented as FormGroup } from '.';
 
 describe('FormGroup', () => {
-  it('matches snapshot', () => {
-    expect(mount(<FormGroup>FormGroup example</FormGroup>)).toMatchSnapshot('FormGroup');
+  it('renders contents without crashing', () => {
```

Comment:
I'm not sure this is what we want by 'convert enzyme to testing library'.

I'd rather take it slower but make sure the tests are written well in the process.

i.e. written from the perspective of the user, not testing implementation details and not giving false code coverage metrics due to vague assertions

Reply:
I hear you, but I'm not sure what else we can test for this component

the FormGroup component essentially just renders content, putting it inside a container.  there's support for an `error` prop which adjusts the display.  that's it.

I guess there's an argument that the second test here that checks for a successful render with the `error` flag set could be deleted, and instead we could/should rely on storybook for coverage of that

Reply:
maybe name the test 'renders contents' and remove 'without crashing'